app/models/kaui/catalog.rb

```diff
@@ -118,6 +118,7 @@ class << simple_plan
           simple_plan.billing_period = plan.billing_period
           simple_plan.trial_length = has_trial ? plan.phases[0].duration.number : 0
           simple_plan.trial_time_unit = has_trial ? plan.phases[0].duration.unit : 'N/A'
+          simple_plan.final_phase = plan.phases.last.type
```

Comment:
This would only say `EVERGREEN` or `FIXEDTERM`? It would be more helpful to have the duration in case of `FIXEDTERM` instead (e.g. here 36 months).

Reply:
Hi @pierre , 
I updated the code, and here is the screenshot with 36 months duration
<img width="1331" alt="image" src="https://github.com/killbill/killbill-admin-ui/assets/80109474/d3a6deeb-5980-4730-b811-169b9d913322">

with unlimited: 
<img width="1331" alt="image" src="https://github.com/killbill/killbill-admin-ui/assets/80109474/b3ede301-86f1-4b7a-a191-e460e91c7155">